examples/alert-configuration/alert-configuration-microsoft-teams.json

```diff
@@ -0,0 +1,102 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "This template creates an encryption at rest on the MongoDB Atlas API, this will be billed to your Atlas account.",
```

Comment:
nit: we can fix this description

Reply:
sadly I didn't notice it, thank you! I've changed also the other alert configuration description